docs/design-proposals/multi-tenancy/multi-tenancy-network.md

```diff
@@ -120,10 +128,118 @@ The reasons that DNS service is per-network instead of per-space are:
 * VPC networks are isolated by nature. It will be difficult to have a shared DNS pod that's accessible to pods in various VPC networks.
 
 ### Service
-For VPC-isolated networking, service IP support has to be provided w/o relying on iptable/IPVS rules on host netns. In other words, traditional kube-proxy can not used in VPC-isolated networking.
 
+#### Type definition
+
+Addition of service spec is metadata.network field. By default, it is the default network of the tenant. 
```

Comment:
"metadata.network" in text also needs to be updated.

Reply:
yes I'll correct it.